dev/ProgressRing/ProgressRing.xaml

```diff
@@ -21,29 +21,55 @@
                     <Grid x:Name="LayoutRoot" Background="Transparent">
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
-                                <VisualState x:Name="Active">
```

Comment:
>Active [](start = 53, length = 6)

Renaming this state has the potential to cause issues for customers who call GoToState("Active"). I think we should leave the default (indeterminate or determinate) as just active to avoid this concern.  @ranjeshj thoughts?

Reply:
Do you have thoughts on this?

---
In reply to: [433548213](https://github.com/microsoft/microsoft-ui-xaml/pull/2584#discussion_r433548213) [](ancestors = 433548213)

Reply:
I'm Still wondering about this @ranjeshj and @karenbtlai

Reply:
I have updated the state names to be `Active`, `InActive`, and `DeterminateActive`.

Reply:
Yes. We should not modify the state names. We could add new states.